src/renderer/ssf-api.ts

```diff
@@ -147,6 +147,13 @@ const throttledRestart = throttle(() => {
   });
 }, 1000);
 
+const throttledSetZoomLevel = throttle((zoomLevel) => {
+  local.ipcRenderer.send(apiName.symphonyApi, {
+    cmd: apiCmds.setZoomLevel,
+    zoomLevel,
+  });
+}, 1000);
```

Comment:
Would it be possible to replace all these delays (1000) by a const please?
Something like ```const DEFAULT_THROTTLE = 1000;```

Reply:
@sbenmoussati 
I updated it.
Thanks!